docs/docker.md

```diff
@@ -0,0 +1,72 @@
+# Running all the components with docker compose
+
+There are 4 docker compose files provided to run origin stack:
+
+-   based on the latest **stable** versions of NPM packages:
+    -   Vanilla Origin version with a **custom Device Registry** (`origin.release.yml`)
+    -   **I-REC compliant** Origin version (`origin.release.irec.yml`)
+-   based on the latest **canary** versions of NPM packages:
```

Comment:
I would just mention the Canary releases, the release version should never be run locally, but only on Heroku/AWS.